viz/charts/linkerd-viz/values.yaml

```diff
@@ -301,7 +321,7 @@ grafana:
   logLevel: ""
   # -- log format (plain, json) of the grafana instance
   # @default -- defaultLogFormat
-  logFormat: ""  
+  logFormat: ""
```

Comment:
I guess the last person who touched this line doesn't have `editorconfig` installed

https://github.com/linkerd/linkerd2/blob/main/.editorconfig#L7